docs/source/design.rst

```diff
@@ -34,6 +34,10 @@ as any errors will affect at most a single request. Though as we describe below
 only processing a single request at a time requires some assumptions about how
 applications are programmed.
 
+The sync worker does not support keep-alive - each connection is closed after
```

Comment:
As a non-native speaker, I'd prefer "persistent connection" instead of keep-alive. On another hand, we already have a  `--keep-alive` setting so perhaps it's OK. @tilgovi, what do you think?